Workbooks/ADXvsLA.json

```diff
@@ -0,0 +1,1430 @@
+            "conditionalVisibility": {
+              "parameterName": "Help",
+              "comparison": "isEqualTo",
+              "value": "Yes"
+            },
+            "name": "Ingestion Text"
+          },
+          {
+            "type": 10,
+            "content": {
+              "chartId": "workbook19356bdc-cfd5-47c7-8e70-05bbd3f39558",
+              "version": "MetricsItem/2.0",
+              "size": 1,
+              "chartType": 2,
+              "resourceType": "microsoft.kusto/clusters",
+              "metricScope": 0,
+              "resourceParameter": "Resources",
+              "resourceIds": [
+                "{Resources}"
+              ],
+              "timeContextFromParameter": "TimeRange",
+              "timeContext": {
+                "durationMs": 2419200000
+              },
+              "metrics": [
+                {
+                  "namespace": "microsoft.kusto/clusters",
+                  "metric": "microsoft.kusto/clusters-Ingestion health and performance-IngestionLatencyInSeconds",
+                  "aggregation": 4,
+                  "splitBy": null
+                }
+              ],
+              "title": "Ingestion Latency ",
+              "showOpenInMe": true,
+              "gridSettings": {
+                "rowLimit": 10000
+              },
+              "sortBy": [],
+              "showExportToExcel": true
+            },
+            "customWidth": "25",
+            "showPin": false,
+            "name": "Ingestion Latency"
+          },
+          {
+            "type": 10,
+            "content": {
+              "chartId": "workbook19356bdc-cfd5-47c7-8e70-05bbd3f39558",
+              "version": "MetricsItem/2.0",
+              "size": 1,
+              "chartType": 2,
+              "resourceType": "microsoft.kusto/clusters",
+              "metricScope": 0,
+              "resourceParameter": "Resources",
+              "resourceIds": [
+                "{Resources}"
+              ],
+              "timeContextFromParameter": "TimeRange",
+              "timeContext": {
+                "durationMs": 2419200000
+              },
+              "metrics": [
+                {
+                  "namespace": "microsoft.kusto/clusters",
+                  "metric": "microsoft.kusto/clusters-Ingestion health and performance-IngestionVolumeInMB",
+                  "aggregation": 1,
+                  "splitBy": null
+                }
+              ],
+              "title": "Ingestion Volume",
+              "showOpenInMe": true,
+              "gridSettings": {
+                "rowLimit": 10000
+              },
+              "sortBy": [],
+              "showExportToExcel": true
+            },
+            "customWidth": "25",
+            "showPin": false,
+            "name": "Ingestion Volume"
+          }
+        ]
+      },
+      "name": "Ingestion ADX",
+      "styleSettings": {
+        "showBorder": true
+      }
+    }
+  ],
+  "fallbackResourceIds": [
```

Comment:
Please remove the reference to Azure resource by making the fallbackResourceIds attribute empty as below. 
**"fallbackResourceIds": [],**

Thanks,
Ruchita